bitnami/grafana-tempo/Chart.yaml

```diff
@@ -28,4 +28,4 @@ name: grafana-tempo
 sources:
   - https://github.com/bitnami/bitnami-docker-grafana-tempo
   - https://github.com/grafana/tempo/
-version: 1.1.6
+version: 1.1.7
```

Comment:
Since it is a feature addition instead of bug fix, I would bump the minor version instead of the patch one

Reply:
> Thanks for this PR, could you please check other charts already using metrics such as the regular grafana in order to replicate the logic/structure followed in the rest of the charts?

Unfortunately I couldn't find any samples of how to deploy this new component, except this one in grafana repo
https://github.com/grafana/tempo/blob/main/operations/kube-manifests/Deployment-metrics-generator.yaml

Reply:
I mean how the metrics-related k8s objects (in the `/templates` folder) are being defined. In the same way, the `metrics:` parameters in the `values.yaml` follow a similar structure in all the Bitnami Helm charts.

Reply:
Sorry, I don't follow what you are trying to say :) 
I've added the templates for new tempo service, `metrics:` parameter in the values.yaml is defined for deploy servicemonitors for services. Maybe you can give me an example?

Reply:
I mean following the same approach used in other Bitnami Helm charts, for example in the Grafana one:
- `values.yaml`: https://github.com/bitnami/charts/blob/7d41614a6e43ff6118a4310a3e72cf230bce94a8/bitnami/grafana/values.yaml#L714
- README: https://github.com/bitnami/charts/blob/master/bitnami/grafana/README.md#metrics-parameters
- Everything related with metrics at templates:
  - https://github.com/bitnami/charts/blob/master/bitnami/grafana/templates/image-renderer-prometheusrules.yaml
  - https://github.com/bitnami/charts/blob/master/bitnami/grafana/templates/image-renderer-servicemonitor.yaml
  - https://github.com/bitnami/charts/blob/master/bitnami/grafana/templates/image-renderer-deployment.yaml
  - https://github.com/bitnami/charts/blob/master/bitnami/grafana/templates/prometheusrules.yaml
  - https://github.com/bitnami/charts/blob/master/bitnami/grafana/templates/servicemonitor.yaml

Reply:
But we do have a metrics parameter in tempo values:
https://github.com/bitnami/charts/blob/master/bitnami/grafana-tempo/values.yaml#L2111
https://github.com/bitnami/charts/tree/master/bitnami/grafana-tempo#metrics-parameters
And the related servicemonitors deploy based on these values
https://github.com/bitnami/charts/blob/master/bitnami/grafana-tempo/templates/compactor/servicemonitor.yaml#L1
https://github.com/bitnami/charts/blob/master/bitnami/grafana-tempo/templates/distributor/servicemonitor.yaml#L1
If you think that we should do it like in Image Renderer in Grafana chart, `.Values.imageRenderer.metrics.enabled` and `.Values.imageRenderer.metrics.serviceMonitor.enabled` than it should be done in every tempo services.